.github/workflows/ci.yml

```diff
@@ -120,3 +120,10 @@ jobs:
       - name: cargo doc
         # We currently skip checking masonry's docs
         run: cargo doc --workspace --all-features --no-deps --document-private-items -Zunstable-options -Zrustdoc-scrape-examples --exclude masonry
+
+  typos:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: crate-ci/typos@v1.21.0
```

Comment:
How often does this update? Do you think it's likely we'll remember to update it manually?

Reply:
It updates at least once a month, but there's really no strong need to bump the version used here as often. I am subscribed to release notifications for it and tend to rerun it on the couple of projects I am contributing to that use it when I get an update, then if it finds a new typo I post a PR and include a version bump.

Reply:
It's also possible to use `@main` but I wouldn't recommend it as it can make CI fail somewhat frequently in annoying ways, plus I've seen it a couple of times that running it right after a release has the GitHub action try to get the latest release which hasn't been built + uploaded yet.

Reply:
This is a good time to bring up Dependabot perhaps.

Reply:
Also, see my similar PR in vello and I have a format that is slightly different than that here.

Reply:
Using `@master` for dictionary updates can introduce failures for unrelated PRs, so similarly as how we update the Clippy version manually, updating this manually makes sense.

Maybe some rarely scheduled batched update by Dependabot can make sense in the future, but for now we can just add this to the checklist of our every-six-weeks CI update where we update the stable Rust version.